components/shared/exploreTabsFilterSort/ExploreTabsFilterSort.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="section pb-0">
+    <h1 class="container mb-5 pb-4 explore-heading">{{ $t('explore') }}</h1>
+    <div v-if="$route.query.search" class="block">
+      {{ $t('general.searchResultsText') }}
+      <span class="text__stroked is-size-3">{{ $route.query.search }}</span>
+    </div>
+    <div class="mb-5 explore-tabs container">
+      <NeoButton
+        class="btn-collection"
+        :selected="selectedTab === tabType.COLLECTION"
+        :label="$t('collections')"
+        @click.native="updateTab(tabType.COLLECTION)" />
+      <NeoButton
+        class="btn-items"
+        :selected="selectedTab === tabType.ITEMS"
+        :label="$t('items')"
+        @click.native="updateTab(tabType.ITEMS)" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+
+enum tabType {
+  COLLECTION = 'collectibles',
+  ITEMS = 'items',
+}
+
+const { $route, $router } = useNuxtApp()
+let selectedTab = ref($route.name?.split('-')[2])
+
+const updateTab = (val) => {
+  selectedTab.value = val
+  $route.query.page = ''
+  let queryOptions: {
+    page: string
+    search?: string | (string | null)[]
+  } = {
+    page: '1',
+  }
+  if ($route.query.search) {
+    queryOptions.search = $route.query.search
+  }
+  $router.replace({
+    path: val,
+    query: queryOptions,
+  })
+}
+</script>
+
+<style lang="scss">
+.explore-heading {
+  font-weight: 700;
+  font-size: 39px;
+}
+
+.explore-tabs {
+  .btn-items {
+    width: 240px;
+    left: -6px;
```

Comment:
🙅

Reply:
I dont think there is any way to make the two buttons meet without any negative margin, since we are trying to make them look like tabs

Reply:
> I dont think there is any way to make the two buttons meet without any negative margin, 

[bulma](https://bulma.io/documentation/elements/button/#button-addons) doc, [buefy](https://buefy.org/documentation/field#addons) doc

> since we are trying to make them look like tabs

why don't we use tab?

Reply:
it wasn't working properly, so we decided to use buttons
CC @yangwao @preschian

Reply:
[oruga](https://oruga.io/components/field.html) doc

Reply:
removed the negative margin